docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -13,103 +13,103 @@ weight: 150
 To send OTel data to Grafana Cloud, refer to [Send data using OpenTelemetry Protocol (OTLP)](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/).
 {{% /admonition %}}
 
-When using the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), you can write metrics into Mimir via two options: `prometheusremotewrite` and `otlphttp`.
+When using the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), you can write metrics into Mimir using two options: `otlphttp` (preferred) and `prometheusremotewrite`.
```

Comment:
I think we should say here we recommend the usage of otlphttp.

The reason why we recommend this is because the exporter is maintained by the community and always up to date with the spec. It's the official way of sending OTLP traffic and we want to recommend the more native path.

Prometheus Remote Write is an exporter in the https://github.com/open-telemetry/opentelemetry-collector-contrib repository donated by the community that is in `beta` stage https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusremotewriteexporter 

We strongly recommend otlphttp. We can leave the example at the bottom of prometheusremotewrite as an experimental section.

Reply:
I'll rework the doc a bit to more strongly favor otlphttp over prometheus remote write.